addons/binding/org.openhab.binding.network/README.md

```diff
@@ -104,8 +104,8 @@ demo.sitemap:
 sitemap demo label="Main Menu"
 {
 	Frame {
-		Switch item=MyDevice
-		Number item=MyDeviceResponseTime
+		Text item=MyDevice
```

Comment:
Did you test that that text widgets really show the current state and not just the item name?